src/pages/BlueSolo/main.ts

```diff
@@ -0,0 +1,96 @@
+import { pageInterface } from '../pageInterface';
+
+export const BlueSolo: pageInterface = {
+  name: 'BlueSolo',
+  domain: 'https://www.bluesolo.org',
+  languages: ['French'],
+  type: 'manga',
+  isSyncPage(url) {
+    return typeof url.split('/')[5] !== 'undefined' && url.split('/')[5] !== '';
+  },
+  sync: {
+    getTitle() {
+      return j
+        .$('div.c-breadcrumb > ol > li:nth-child(3) > a')
+        .text()
+        .trim();
+    },
+    getIdentifier(url) {
+      return utils.urlPart(url, 4);
+    },
+    getOverviewUrl() {
+      return utils.absoluteLink(j.$('div.c-breadcrumb > ol > li:nth-child(3) > a').attr('href'), BlueSolo.domain);
+    },
+    getEpisode(url) {
+      const type = url.split('/')[5];
+      if (type.includes('chapitre')) {
+        return Number(type.replace('chapitre-', ''));
+      }
+      if (type.includes('one-shot')) {
+        return 1;
+      }
+      return NaN;
+    },
+    nextEpUrl() {
+      const nextChapter = j
+        .$(
+          '.header > .wp-manga-nav > .select-view > .chapters_selectbox_holder > .selectpicker_chapter > label > select > option:selected',
```

Comment:
This selector is too specific.
I think `.selectpicker_chapter select option:selected` should be enough

Reply:
Yeah it's enough